internal/kube/client.go

```diff
@@ -82,7 +82,7 @@ func (c *client) Octopus() octopus.OctopusInterface {
 func (c *client) IsPodDeployed(namespace, name string) (bool, error) {
 	_, err := c.Static().CoreV1().Pods(namespace).Get(name, metav1.GetOptions{IncludeUninitialized: true})
 	if err != nil {
-		if strings.Contains(err.Error(), "not found") {
+		if strings.Contains(err.Error(), "Not found") {
```

Comment:
That won't work. The error messages coming from a kubernetes API are in lower case. Either leave it in lower case or all checks should be wrapped with `strings.ToLower()`

Reply:
got it - I will change it :)